deployment/ansible/roles/model-my-watershed.app/meta/main.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
   - { role: "azavea.pip" }
   - { role: "azavea.nodejs" }
   - { role: "azavea.phantomjs" }
+  - { role: "azavea.build-essential" }
```

Comment:
This needs to also be specified in `deployment/ansible/roles.yml`. It's probably being pulled in as a transitive dependency.

Reply:
https://github.com/WikiWatershed/model-my-watershed/blob/aebaf8ce027abe323a2e1384a1e6f823ac2dc31e/deployment/ansible/roles.yml#L41-L42 ?